### DISCUSSION TITLE
Starting point gas price

### DISCUSSION THREAD

**Discussion by Mrooie** [coccoinomane/crabada.py#58] [Q&A] [answered]
Hi!
Is it possible to set a starting gas fee, which will be the limit for sending teams for mining?
Today Avalanche was minting the roostr and gas was over 4000 gwei, so about 0,9AVAX. 🙈  That would be a killer if bot would start mining at this time.

**Comment by loayei** [comment]
There is a gas limit option. Check the README

**Reply by Mrooie** [reply]
But it is for reinforcing only, right?
I'm talking about >starting a mine< gas limit - at the moment that bot sends teams for mining. Not later, when reinforcing - this one works great.

**Reply by Mrooie** [reply]
Ah, I forgot about that post!
Yeah, it was introduced next day and it works perfect. ;)
Sorry for not deleting this thread!

**Reply by coccoinomane** [reply]
No need to delete the thread, it can be useful for other players :-)

**Comment by coccoinomane** [comment, marked as answer]
Hi @Mrooie !

The parameters you are looking for are `USER_1_MINE_MAX_GAS` and `USER_1_CLOSE_MINE_MAX_GAS` and have been kindly introduced a few days ago by @EugeneTeu in https://github.com/coccoinomane/crabada.py/pull/60 🙂

Let us know if it works for you!

Cheers,
Cocco
